src/app/widgets/notifications/NotificationsGrid.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace app\widgets\notifications;
+
+use yii\base\Widget;
+
+class NotificationsGrid extends Widget
+{
+    public $dataProvider;
+    public $columns;
+
+    public function init() {
+        parent::init();
+        $this->dataProvider = $this->dataProvider->getModels();
+        $this->columns = $this->columns;
+    }
+    
```

Comment:
te dejo dos links para el tema de los assets que hablamos
https://stackoverflow.com/questions/34021875/how-define-widget-assets-file-in-yii2
https://www.yiiframework.com/doc/guide/2.0/es/structure-assets